website.py

```diff
@@ -199,7 +204,7 @@ class User(db.Model):
     #: Private key, for first-time access without password
     privatekey = db.Column(db.String(22), nullable=False, unique=True, default=newid)
     #: Public UID; not clear what this could be used for
-    uid = db.Column(db.String(22), nullable=False, unique=True, default=newid)
+    uid = db.synonym('uuid')
```

Comment:
Since a string isn't a synonym for a UUID, drop this synonym. Change it in the code wherever it refers to `uid`. You can use `buid` or `suuid` (your choice) wherever a UUID needs to appear in URLs.

Since we don't care about `uid` values exported to Mailchimp anymore, we don't need to worry about creating a new `uid` property here (the encoding scheme is distinct from both `buid` and `suuid`).